Formula/monkeysphere.rb

```diff
@@ -48,6 +53,35 @@ def install
     # Consequently, this is a bit of a naughty hack but the least worst option.
     inreplace pkgshare/"keytrans", "#!/usr/bin/perl -T",
                                    "#!/usr/bin/perl -T -I#{libexec}/lib/perl5"
+
+    # find is used with GNU only permissions mode
+    inreplace pkgshare/"common", "$(find ",
+                                 "$(gfind "
+    # use liblockfile
+    inreplace pkgshare/"common", "lockfile-create",
```

Comment:
These should have patches upstreamed.

Reply:
I have already reported these issues upstream. I reported them in their irc channel since their bug tracker seemed to be dead.